### PR TITLE
fix: link reference in architecture

### PR DESCRIPTION
**Pull request by sakshi1215**
Signed-off-by: Sakshi Patle <sakshi.patle121@gmail.com>
fixes https://github.com/adoptium/ci-jenkins-pipelines/issues/447

### CODE REVIEW

**Comment by sakshi1215** [comment]
@adamfarley @smlambert please review

File: docs/ARCHITECTURE.md [hunk 1]
```diff
@@ -17,13 +17,13 @@ A rough structure of the build is as follows
...
+These are the first scripts run by the top level jobs. Which in turn invoke the downstream build jobs for each configuration/platform, deploying the results to GitHub. The majority of the code that defines this stage is contained within [build_base_file.groovy](https://github.com/adoptium/ci-jenkins-pipelines/blob/master/pipelines/build/common/build_base_file.groovy).
 
 #### Individual Build Jobs
 
 These are invoked by the parent pipeline job and each build creates the archives/installers for a given platform. The majority
-of this code is defined in [openjdk_build_pipeline.groovy](https://github.com/adoptium/temurin-build/blob/master/pipelines/build/common/openjdk_build_pipeline.groovy).
+of this code is defined in [openjdk_build_pipeline.groovy](https://github.com/adoptium/ci-jenkins-pipelines/blob/master/pipelines/build/common/).
```
Comment (zdtsw) [inline docs/ARCHITECTURE.md:29]: any reason to remove openjdk_build_pipeline.groovy from the change?

**Comment by adamfarley** [comment]
In #447 I advised using a relative path to prevent this issue happening in the future. 

Another (better?) option would be to replace everything above the top-level directory with a forward slash (so https://github.com/user/repo/folder/folder2/filename.txt becomes /folder/folder2/filename.txt).

**Review by adamfarley — CHANGES_REQUESTED** [review]
Ditto zdtsw's comment. Also, in https://github.com/adoptium/ci-jenkins-pipelines/issues/447 I advised using a relative path to prevent this issue happening in the future.

Another (better?) option would be to replace everything above the top-level directory with a forward slash (so https://github.com/user/repo/folder/folder2/filename.txt becomes /folder/folder2/filename.txt).

**Comment by sakshi1215** [comment]
> Ditto zdtsw's comment. Also, in #447 I advised using a relative path to prevent this issue happening in the future.
> 
> Another (better?) option would be to replace everything above the top-level directory with a forward slash (so https://github.com/user/repo/folder/folder2/filename.txt becomes /folder/folder2/filename.txt).

I thought it is not a good practice to use relative paths for the actively changing repositories, I prefer relative path for docs repositories where you know you won't be messing around with the structure quiet often. But for code absolute path seems more declarative as well as is more descriptive the next time you make changes in your code directories. I may be wrong too but just laying out my opinions @adamfarley .

**Comment by adamfarley** [comment]
> > Ditto zdtsw's comment. Also, in #447 I advised using a relative path to prevent this issue happening in the future.
> > Another (better?) option would be to replace everything above the top-level directory with a forward slash (so https://github.com/user/repo/folder/folder2/filename.txt becomes /folder/folder2/filename.txt).
> 
> I thought it is not a good practice to use relative paths for the actively changing repositories, I prefer relative path for docs repositories where you know you won't be messing around with the structure quiet often. But for code absolute path seems more declarative as well as is more descriptive the next time you make changes in your code directories. I may be wrong too but just laying out my opinions @adamfarley .

Sounds sensible. Good reasoning. :)

Looks like we'll be good to go once you've resolved zdtsw's comment.

File: docs/ARCHITECTURE.md (hunk 1 above)
Comment (gdams) [inline docs/ARCHITECTURE.md:29]:
```suggestion
of this code is defined in [openjdk_build_pipeline.groovy](https://github.com/adoptium/ci-jenkins-pipelines/blob/master/pipelines/build/common/openjdk_build_pipeline.groovy).
```

**Review by gdams — APPROVED** [review]

File: docs/ARCHITECTURE.md [hunk 2]
```diff
@@ -17,13 +17,13 @@ A rough structure of the build is as follows
 #### Regenerate Jobs
 
 Jenkins CI jobs are generated by the Jenkins [regeneration pipelines](https://ci.adoptium.net/job/build-scripts/job/utils/) and placed into the <https://ci.adoptium.net/job/build-scripts/> location. These builds coordinate the entire process of nightly and release builds.
-Documentation for this process can be found at [regeneration docs](https://github.com/adoptium/temurin-build/tree/master/pipelines/build/regeneration)
+Documentation for this process can be found at [regeneration docs](https://github.com/adoptium/i-jenkins-pipelines/tree/master/pipelines/build/regeneration)
```
Comment (karianna) [inline docs/ARCHITECTURE.md:20]:
```suggestion
Documentation for this process can be found at [regeneration docs](https://github.com/adoptium/ci-jenkins-pipelines/tree/master/pipelines/build/regeneration)
```

**Review by karianna — APPROVED** [review]

**Review by adamfarley — APPROVED** [review]
LGTM. Merging.

**Pull request merged**